gen-bundle-image.sh

```diff
@@ -69,7 +69,7 @@ function generate_manifest {
 function sign_bundle_files {
   local preset=$1
   rm -fr *.sig
-  if [[ ${preset} = "podman" ]]; then
+  if [[ ${preset} != "okd" ]]; then
```

Comment:
Fwiw, I would change this logic to check if `vfkit_bundle_arm64` is empty or not, this way we don't have multiple places in this script which need to know which bundles support arm64, we set `vfkit_bundle_arm64` according to preset, and the rest of the code does the right thing once this is set or not.